src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule32packagestatement/InputPackageStatement.java

```diff
@@ -0,0 +1,8 @@
+package com.google.checkstyle.test. // violation 'package statement should not be line-wrapped.'
+        chapter3filestructure.rule32packagestatement;
+
+public class InputPackageStatement {
+  // Long line ---------------------------------------------------------------------------------------- // violation 'Line is longer than 100 characters (found 166).'
```

Comment:
can we do `violation below|above`
we did such options to allow not violate long lines.

Reply:
done

https://github.com/checkstyle/checkstyle/blob/c0d7a210d3c0fcfafc4242462a88a1006923cd68/src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule32packagestatement/InputPackageStatement.java#L5-L9